### DISCUSSION TITLE
Energy Dashboard not populating

### DISCUSSION THREAD

**Discussion by ajlennon** [BottlecapDave/HomeAssistant-OctopusEnergy#835] [General]
Hi all,

Fantastic plugin! Thanks for doing this. I'm an experienced embedded engineer but have only recently installed HA so apologies if this is a stupid question. I have been through the FAQ as suggested but can't spot anything obvious...

I've added the Octopus integration and connected it up to my Octopus account. I'm on the Agile tariff. I see the correct data coming through on pricing and so forth to the electricity and gas sensors.

[Edit: NB. I should add I've asked Octopus for a "mini" but am currently waiting on this and using the API]

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/12e33f72-be0a-4b1f-85c0-7e90b83eb88f)

I went through the documentation on configuring the Energy dashboard and left it overnight to update itself but I don't see anything coming through

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/023fe13f-e046-4fa2-ab27-571528461857)

The configuration for electricity and gas is like this

- Electricity 23J0205110 1300060515133 Previous Accumulative Consumption
- Electricity 23J0205110 1300060515133 Previous Accumulative Cost
- Gas E6S13726392361 1664288707 Previous Accumulative Consumption (kWh)
- Gas E6S13726392361 1664288707 Previous Accumulative Cost

[Edit: I made sure to use the octopus sensors as indicated in **bold** so I *think* I got this right?]

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/a47e77b0-7e08-4abe-b054-f7eea7819afc)

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/1fb222cc-7af9-4be0-a907-1f87b5885431)

I've enabled "debug logging" but can't spot any logs yet.

I assume I am missing something obvious here but could you perhaps signpost me to what I should be looking at?

Thanks!

Alex

**Comment by BottlecapDave** [comment]
Hello, no such thing as a stupid question.

Without the mini, the integration is subject to the delay in OE getting the information from your meter. This means that previous consumption data is usually at least a day out and isn't available at the stroke of midnight. By default, the integration looks at data for the day before, but if your meter takes longer to report its data, you might need to adjust this to more via the integration configuration. I can't see the day that you're looking at on your dashboard, but assuming that your data arrives the following day, the data should be available available up to 14/04/2024.

> [Edit: I made sure to use the octopus sensors as indicated in bold so I think I got this right?]

The sensors you want are not in bold in the docs - you want the sensors in the form `octopus_energy:gas_{{METER_SERIAL_NUMBER}}_{{MPRN_NUMBER}}_previous_accumulative_consumption`, not `sensor.octopus_energy_gas_{{METER_SERIAL_NUMBER}}_{{MPRN_NUMBER}}_previous_accumulative_consumption` (please note the subtle difference in naming)

**Comment by ajlennon** [comment]
>Hello, no such thing as a stupid question.

Thanks for the kind words. Yes you're absolutely right I got confused. I did follow the instructions and had set the octopus_ prefix not the sensor_octopus_ prefix (I think). 

I've just changed these. Is there any way to make the energy config boxes bigger as it's super hard to work out what's what? (I understand from the FAQ why the sensor names need to be as they are)

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/8e4f43f5-c5be-4dba-8b0b-df947aaae773)

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/5ce49682-2948-4ea5-bafa-68503474c9a2)

**Reply by BottlecapDave** [reply]
Unfortunately the size of the boxes is a HA thing :( Would be good if they took advantage of available real estate.

**Comment by ajlennon** [comment]
WHOAH I just got yesterday's stuff! I get it - I was being stupid of course - with the API I can only get yesterday's figures. AWESOME

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1537834/51a1b081-da3d-48e9-9767-00bbe7db007e)

**Reply by BottlecapDave** [reply]
Glad it's coming through
